packages/core/src/common/error/errors/custom-error.ts

```diff
@@ -4,11 +4,6 @@ export default interface CustomError extends Error {
     subtype?: string;
 }
 
-export function isCustomError(error: any): error is CustomError {
-    return (
-        typeof error.message === 'string' &&
-        typeof error.type === 'string' &&
-        (typeof error.subtype === 'string' || !error.subtype) &&
-        error instanceof Error
-    );
+export function isCustomError(error: unknown): error is CustomError {
+    return typeof error === 'object' && error !== null && 'message' in error && 'type' in error;
```

Comment:
I see `subtype` checking is removed, is this intentional?

Reply:
Yes, intentional since not all custom errors have a subtype from what I found.  We will need to type this properly later.